libshvchainpack/src/chainpack/rpcvalue.h

```diff
@@ -484,6 +489,11 @@ class SHVCHAINPACK_DECL_EXPORT RpcValue
 	[[deprecated("Use asMap instead")]] const Map &toMap() const { return asMap(); }
 	[[deprecated("Use asIMap instead")]] const IMap &toIMap() const { return asIMap(); }
 
+	template<typename T> T to() const
+	{
+		return rpcvalue_cast<T>(*this);
```

Comment:
Možná by bylo lepší dát tuhle funkci jako tu hlavní implementaci a rpcvalue_cast označit jako deprecated? Co mylíš @fvacek

Reply:
Hm, tam mas pravdu. Ja to zkousel specializovat primo v te tride. Takhle me to nenapadlo. Je zajimavy ze i na webu jsem vsude nasel ze to nejde ... :-)
Tak v tom pripade je to na vas, podle me to takhle staci, ale nemam nic proti tomu to predelat. Akorat to nebude zpetne kompatibilni.

Reply:
Bude, jen bude třeba změnit všechny ty specializace. na rpcvalue_cast se akorát nasadí deprecated.

Ale jinak se to klidně může zmergovat a já to pak udělám :)